src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -167,4 +218,43 @@ public JsonWebKey supplyKey(
 
         return keysByKid;
     }
+
+    private final class OAuthRealmObject
+    {
+        private static final int MAX_SCOPES = 48;
+
+        private final Map<String, Long> scopeBitsByName = new CopyOnWriteHashMap<>();
+
+        private long realmBit;
+        private long nextScopeBitShift;
+
+        private OAuthRealmObject(long realmBit)
```

Comment:
Constructor parameter should be on next line, not sure why checkstyle is not flagging this already.